scripts/hotspot.js

```diff
@@ -340,4 +341,13 @@
     };
   };
 
+  /**
```

Comment:
Added JSDocs

Reply:
Yup :-D I created a PR to your repository in order to add a `returns` declaration as well.